### PR TITLE
Multi context refactor

### PR DESCRIPTION
**Pull request by Revolyssup**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: internal/graphql/schema/schema.graphql [hunk 1]
```diff
@@ -401,9 +415,6 @@ type Query {
 #
 
 type Mutation {
-  # Change the Addon Status
-  changeAddonStatus(input: AddonStatusInput): Status!
-
```
Comment (Abhishek-kumar09) [inline internal/graphql/schema/schema.graphql:406]: This Mutation is removed intentionally?

File: internal/graphql/schema/schema.graphql [hunk 2]
```diff
@@ -99,9 +100,6 @@ type AddonList {
 
   # Owner
   owner: String!
-
-  # Endpoint (if applicable)
-  endpoint: String!
```
Comment (Abhishek-kumar09) [inline internal/graphql/schema/schema.graphql:104]: This one too?

File: internal/graphql/schema/schema.graphql (hunk 1 above)
Comment (Abhishek-kumar09) [reply internal/graphql/schema/schema.graphql:406]: I am removing this query usage from the UI...

**Comment by MUzairS15** [comment]
@Revolyssup `Flush MeshSync data` is not still context-specific, though GraphQL Schema is changed, server is not utilising it.

**Comment by Revolyssup** [comment]
> @Revolyssup `Flush MeshSync data` is not still context-specific, though GraphQL Schema is changed, server is not utilising it.

Yups thanx for pointing it out. I am making changes rii

**Comment by Revolyssup** [comment]
> @Revolyssup `Flush MeshSync data` is not still context-specific, though GraphQL Schema is changed, server is not utilising it.

I wont be dropping table for soft reset now. Will just delete data for the specific cluster

**Comment by MUzairS15** [comment]
> > 
> 
> SInce the hard reset deletes the database, remember to keep it global. I am propogating context only to soft reset

Yes

**Review by Abhishek-kumar09 — APPROVED** [review]

**Pull request merged**